src/main/java/seedu/teachbook/logic/commands/AddClassCommand.java

```diff
@@ -23,7 +23,8 @@
             + PREFIX_NAME + "classobject A ";
 
     public static final String MESSAGE_SUCCESS = "New class added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This class already exists in the teach book";
+    public static final String MESSAGE_DUPLICATE_CLASS = "This class already exists in the teachbook book";
+
```

Comment:
I'm not sure whether you pull before changing variable name haha, because I change the "teachbook book" to "teach book". Or you just think it should be "teachbook book"?